src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotCompiledCodeStream.java

```diff
@@ -1029,6 +1033,10 @@ private boolean isNarrowOop(Value oopValue) {
         return oopValue.getPlatformKind() != runtime.getHostJVMCIBackend().getTarget().arch.getWordKind();
     }
 
+    private boolean isVector(Value vectorValue) {
```

Comment:
`vectorValue` -> `value` (otherwise the name seems to presume a true return value)

Reply:
done